vite-project-kabinet/src/Content/CardFullArticle.jsx

```diff
@@ -2,6 +2,9 @@ import { useState } from "react"
 import "./article.style.css"
 import { useEffect } from "react"
 import { useLocation } from "react-router"
+import { React } from "react"
+import { format } from "date-fns"
```

Comment:
Tohle taky chybí v `package.json`, nejde to pak pustit.

Reply:
Ako je to možné? Že by som to inštalovala bez --save? Je pravda, že date-fns nevidím ani v node_modules. Keď si zadám do terminálu npm ls date-fns, tak mi vypíše toto:

vanilla-starter-kit@1.0.0 /Users/borka
└── date-fns@4.1.0

Reply:
Podle mne si to prostě nenainstalovala. A npm funguje tak nešikovně, že pokud ten balíček nainstaluje jako svoji závislost nějaká jiná knihovna, tak to bude fungovat, protože to tam je uložené. Ale já jsem třeba `vanilla-starter-kit` neinstaloval, takže to nemám.

Ale je to jednoduché... cokoli co Ty přímo používáš musí být v package.json. Pokud není, je třeba to přidat přes `npm install`

Reply:
S týmto mám trochu problém - po inštalácii sa mi to nezobrazuje v závislostiach package.json.

Inštalujem podľa dokumentácie knihovny date-fns:
npm install date-fns --save

Nepomohlo ani to odinštalovať a nainštalovať to znovu.

Nepomohlo ani vložiť to priamo do súboru package.json a potom spustiť npm install - prebehlo to, ale v node_modules to nie je.
"dependencies": {
  "date-fns": "^4.1.0"
}

Nenapadá ťa náhodou, že v čom by mohol byť problém?

Reply:
Pushla som date-fns dependency do package.json. Ide ti to teraz spustiť?

Reply:
A v jaké jsi složce když voláš to `npm install date-fns`? Nejsi náhodou v rootu gitu?

Reply:
Jo, bolo to tým, díky. Už by ti to malo ísť spustiť.